### PR TITLE
Adding documention submitted by "Balestrini, Santiago" <Santiago.Balestrini@gtri.gatech.edu>

### PR DESCRIPTION
**Pull request by HuiJun**

### CODE REVIEW

File: docs/Windows_Installation_Instructions.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
+# Instructions for Windows
+
+This document describes how to configure and start up a running version of MMS 4.
+
+Versions Used to Test these instructions
+* [MMS v4.0.0-a2](https://github.com/Open-MBEE/mms/tree/4.0.0-a2)
```
Comment (sanbales) [inline docs/Windows_Installation_Instructions.md:6]: There is a newer image now `[MMS v4.0.0-b2](https://github.com/Open-MBEE/mms/tree/4.0.0-b2)`

File: docs/Windows_Installation_Instructions.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+1. Clone the `mms` repo
+
+    ```bash
+    git clone git@github.com:Open-MBEE/mms.git
+    ```
+
+1. Checkout the right branch/tag:
+
+    ```bash
+    git checkout 4.0.0-a2
```
Comment (sanbales) [inline docs/Windows_Installation_Instructions.md:50]: `git checkout 4.0.0-b2`

File: docs/Windows_Installation_Instructions.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+    docker-compose up -d
+    ```
+
+    You should see something like this:
+
+    ```bash
+    Creating network "mms_default" with the default driver
+    Creating mms_postgres_1      ... done
+    Creating mms_elasticsearch_1 ... done
+    Creating mms                 ... done
```
Comment (sanbales) [inline docs/Windows_Installation_Instructions.md:84]: Not super important but we may need to add a new line for `minio`, something like: `    Creating mms_minio_1 ... done` (double checking that is what it looks like when cloning from the new `4.0.0-b2` tag.

File: docs/Windows_Installation_Instructions.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+    ```
+
+    You should see something like this:
+
+    ```bash
+           Name                      Command               State                       Ports
+    -------------------------------------------------------------------------------------------------------------
+    mms                   java -Djdk.tls.client.prot ...   Up      0.0.0.0:8080->8080/tcp
+    mms_elasticsearch_1   /usr/local/bin/docker-entr ...   Up      0.0.0.0:9200->9200/tcp, 0.0.0.0:9300->9300/tcp
+    mms_postgres_1        docker-entrypoint.sh postgres    Up      0.0.0.0:5432->5432/tcp
```
Comment (sanbales) [inline docs/Windows_Installation_Instructions.md:104]: Again, not super important, but maybe we need to add one line for `minio`:

`mms_minio_1             /usr/bin/docker-entrypoint ...   Up      0.0.0.0:9000->9000/tcp`

File: docs/Windows_Installation_Instructions.md [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+### Check out the REST API
+
+1. Go to [`http://localhost:8080/v3/swagger-ui.html`](http://localhost:8080/v3/swagger-ui.html) to see more info on the API.
+
+1. Expand the `POST /authentication` accordion and click on `Try it out`
+
+1. Set the Request body to be:
+    > The values below are based on the `application-test.properties` configuration file that was copied above, and the values for: `sdvc.admin.username` and `sdvc.admin.password`, if you changed that, make sure the values below match what you put in the application file.
+
+    ```json
```
Comment (sanbales) [inline docs/Windows_Installation_Instructions.md:145]: The new version allows you to authenticate within the modal now, so no need to copy-paste the token.  @dlamoris would you like me to update the instructions to reflect that?

**Comment by sanbales** [comment]
I can prepare an update set of instructions, but FYSA, when I tried to do this again from the MMS server it would fail when trying to build the Docker image.  It worked fine when using the MMSRI repo though ('cause the image is already uploaded).

Maybe we want to move these instructions to the MMSRI repo?  Obviously, I'll defer to what y'all want to do.

This is the error I get when trying to build the docker image:
```
> Task :cameo:processResources NO-SOURCE
> Task :cameo:classes
> Task :cameo:jar
> Task :core:processResources
> Task :core:classes
> Task :core:jar
> Task :crud:processResources NO-SOURCE
> Task :crud:classes
> Task :crud:jar
> Task :data:processResources NO-SOURCE
> Task :data:classes
> Task :data:jar
> Task :search:compileJava
> Task :elastic:compileJava
> Task :elastic:compileJava FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':elastic:compileJava'.
> Could not resolve all files for configuration ':elastic:compileClasspath'.
   > Could not download commons-logging-1.1.3.jar (commons-logging:commons-logging:1.1.3)
      > Could not get resource 'https://repo.spring.io/plugins-release/commons-logging/commons-logging/1.1.3/commons-logging-1.1.3.jar'.
         > Could not GET 'https://repo.spring.io/plugins-release/commons-logging/commons-logging/1.1.3/commons-logging-1.1.3.jar'. Received status code 401 from server: Unauthorized

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 3m 5s
```

**Comment by HuiJun** [comment]
Closing this PR, since these docs were added to MMSRI with updates

**Pull request closed without merging**